go/libraries/doltcore/doltdb/root_val.go

```diff
@@ -87,6 +87,8 @@ type RootValue interface {
 	GetTableHash(ctx context.Context, tName TableName) (hash.Hash, bool, error)
 	// GetTableSchemaHash returns the hash of the given table's schema.
 	GetTableSchemaHash(ctx context.Context, tName TableName) (hash.Hash, error)
+	// GetTableNames retrieves the names of all tables and root objects for a RootValue across all schemas.
```

Comment:
```suggestion
	// GetAllTableNames retrieves the names of all tables and root objects for a RootValue across all schemas.
	// Note that |includeRootObjects| is ignored since root objects are not used directly in Dolt. 
```